awscli/testutils.py

```diff
@@ -639,11 +606,8 @@ def aws(command, collect_memory=False, env_vars=None,
     """
     if platform.system() == 'Windows':
         command = _escape_quotes(command)
-    if 'AWS_TEST_COMMAND' in os.environ:
-        aws_command = os.environ['AWS_TEST_COMMAND']
```

Comment:
Just checking but is this environment variable used in any of the tests? I did a quick grep and did not see anything so I think we should be good.